pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -290,11 +295,12 @@ func (p *PolicyData) EnsureRBACPolicy() genericapiserver.PostStartHookFunc {
 					if err != nil {
 						// don't fail on failures, try to create as many as you can
 						utilruntime.HandleError(fmt.Errorf("unable to reconcile rolebinding.%s/%s in %v: %v", rbac.GroupName, roleBinding.Name, namespace, err))
+						someReconciliationFailed = true
 					}
 				}
 			}
 
-			return true, nil
+			return !someReconciliationFailed, nil
```

Comment:
if reconciliation failed, can we log something at the end before doing `return !someReconciliationFailed, nil` so it is clear from the log which reconciliation failures occurred in the last run? (currently, all the attempts blend together and it's hard to tell the particular reconciliations that caused the ultimate failure)